public/js/components/AtomEdit/CustomEditors/AudioEditor.js

```diff
@@ -0,0 +1,52 @@
+import React, { PropTypes } from 'react';
+import {atomPropType} from '../../../constants/atomPropType';
+import {ManagedField, ManagedForm} from "../../ManagedEditor";
+import FormFieldTextInput from "../../FormFields/FormFieldTextInput";
+import FormFieldImageSelect from '../../FormFields/FormFieldImageSelect';
+import FormFieldNumericInput from "../../FormFields/FormFieldNumericInput";
+
+export class AudioEditor extends React.Component {
+
+  static propTypes = {
+    atom: atomPropType.isRequired,
+    onUpdate: PropTypes.func.isRequired,
+    onFormErrorsUpdate: PropTypes.func
+  };
+
+  render () {
+    return (
+      <div>
+        <ManagedForm formName="audioEditor" data={this.props.atom} updateData={this.props.onUpdate} onFormErrorsUpdate={this.props.onFormErrorsUpdate} >
+          <ManagedField fieldLocation="title" name="Title">
+            <FormFieldTextInput key={1}/>
```

Comment:
Do you need those `key` attributes? React ... 🤦‍♂️

Reply:
Oh good point. I can at least use an index. Will check that out